<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication1Class</class>
 <widget class="QWidget" name="QtWidgetsApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1725</width>
    <height>1213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication1</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1441</width>
     <height>1071</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>1130</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>100</y>
     <width>141</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="text">
     <string>打开图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="text">
     <string>图像旋转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="text">
     <string>图像hsv变化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="text">
     <string>图像锐化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="text">
     <string>图像像素随机模糊</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>320</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>播放视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>370</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>随机模糊</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
